src/lib/utils.ts

```diff
@@ -86,12 +86,16 @@ export const fetchTokensPrices = async () => {
 export const fetchLatestL1L2Transactions = async (): Promise<
```

Comment:
Fix the conflict in `@/lib/utils.ts` so this can be merged.

Reply:
> Fix the conflict in `@/lib/utils.ts` so this can be merged.

Hi @saimeunt, changes applied 🫡